worker-languagedetection-container/pom.xml

```diff
@@ -151,11 +151,13 @@
                         <input.folder>${test.data}</input.folder>
                         <expected.folder>${test.data}</expected.folder>
                         <datastore.enabled>false</datastore.enabled>
+                        <worker.testing.usehttpdatastore>true</worker.testing.usehttpdatastore>
                     </systemPropertyVariables>
                     <environmentVariables>
                         <CAF_APPNAME>test/worker-languagedetection</CAF_APPNAME>
                         <CAF_CONFIG_PATH>${test.configs}</CAF_CONFIG_PATH>
                         <CAF_INTEGRATIONTESTS_FS_PATH>\\${docker.host.address}@${webdav.apache.port}\webdav</CAF_INTEGRATIONTESTS_FS_PATH>
```

Comment:
Is there more to do here?

Reply:
I don't think so. I didn't remove `CAF_INTEGRATONTESTS_FS_PATH` as it is read by the `cfg~test~worker-languagedetection~FileSystemDataStoreConfiguration` configuration file used by the worker during the integration tests (not the integration tests themselves): 

```
{
    "dataDir": "${CAF_INTEGRATIONTESTS_FS_PATH:-/srv/common/webdav}"
}
```